logstash-core/lib/logstash/inputs/metrics.rb

```diff
@@ -27,8 +32,8 @@ def stop
       LogStash::Instrument::Collector.instance.delete_observer(self)
     end
 
-    def update(time, snapshot)
-      @logger.debug("Metrics input: received a new snapshot", :snapshot => snapshot, :event => snapshot.to_event) if @logger.debug?
+    def update(time_of_snapshot, snapshot)
```

Comment:
why this renaming? I guess a better name might be ```timestamp```. I mean this because now you made the var dependant on the content of the second parameter, if we call this ```timestamp```, this var would have a general meeting, time of the update. Also if this is the time of the snapshot, can this be included in the snapshot ?